addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20720,13 +20720,19 @@ msgctxt "#36631"
 msgid "Force weak SMBv1 security for compatibility with the USB sharing features on some WiFi routers and NAS devices."
 msgstr ""
 
-#empty strings from id 36632 to 36898
 #. Text in confirm dialog when enabling setting "Services -> Control -> Allow remote control via HTTP"
 #: xbmc/network/NetworkServices.cpp
 msgctxt "#36632"
 msgid "Anyone who has access to the web interface will be able to completely control this application and, therefore, this device. You should protect the interface with a strong password that must be set below. Proceed?"
 msgstr ""
 
+#. Text in confirm dialog when enabling setting "Services -> Control -> Allow remote control from applications on other systems"
+#: xbmc/network/NetworkServices.cpp
+msgctxt "#36633"
+msgid "These services offer neither authentication nor encryption. Anyone that can connect to them will be able to completely control this application and, therefore, this device, so they should never be exposed on the Internet. Proceed?"
```

Comment:
lowercase `internet` at the end?

Reply:
Internet (with capital I) is correct and also used in all other places in `strings.po`.